application/heartbeater.go

```diff
@@ -56,11 +57,11 @@ func (hb *heartbeater) Close() {
 func (hb *heartbeater) beat() {
 	tags := hb.application.Map()
 	tags["component"] = "wavefront-generated"
-	hb.send(tags)
+	hb.Send(tags)
```

Comment:
Revert this? The internal function is lowercase `send`. Why is this capitalized?